client/app/scripts/actions/app-actions.js

```diff
@@ -1,28 +1,23 @@
 import debug from 'debug';
-import find from 'lodash/find';
+import { find } from 'lodash';
```

Comment:
I haven't tested it myself, but it shouldn't matter anymore now that we're using Webpack 2: https://webpack.js.org/guides/tree-shaking/ :)